### PR TITLE
Remove double unescape path variable

### LINKED ISSUE #195: Bad request returned for percent-encoded get_by_attribute
## Current Behavior
https://rest.fs.neo.org/7ZVSQHtioVRiDT2btN9xEmQhczXV9vryCEAHUawfD8Sc/7-1712215152/index.html#suites/4f5ca4977289636df3b53e1debc15dcf/84697d2930280bd1/

> Exception: Failed to get object via HTTP gate:
                request: /v1/get_by_attribute/A9pnctophx5PFt4eUd8d2aa6otjgnQZNzyvt3HCeygFD/cat%25jpeg/cat%25jpeg,
                response: {"message":"Invalid format for parameter attrKey: error unescaping path parameter 'attrKey': invalid URL escape \"%jp\""}
,
                status code: 400 Bad Request

## Expected Behavior
No error, the URL seems to be fine here.

### PR DESCRIPTION
**Pull request by smallhive**
Closes #195.

### CODE REVIEW

File: spec/rest.yaml [hunk 1]
```diff
@@ -1032,8 +1032,23 @@ paths:
...
+                type: string
+                example: attributename
+        - name: attrVal
+          in: path
+          description: Object attribute value to match.
+          required: true
+          content:
+            plain/text:
+              schema:
+                type: string
```
Comment (tatiana-nspcc) [inline spec/rest.yaml:1051]: I believe it's better to edit $ref: '#/components/parameters/attrKey' than to copy-paste the same piece multiple times.

**Comment by tatiana-nspcc** [comment]
Have you checked that the test `Test Put over REST, Get over REST with headers` from `neofs-testcases` works with attributes `{'cat%jpeg': 'cat%jpeg'}`?

File: spec/rest.yaml [hunk 2]
```diff
@@ -1032,8 +1032,23 @@ paths:
       operationId: getByAttribute
       parameters:
         - $ref: '#/components/parameters/containerId'
-        - $ref: '#/components/parameters/attrKey'
-        - $ref: '#/components/parameters/attrVal'
+        - name: attrKey
+          in: path
+          description: Asdasdas attribute key to search.
```
[the remark this answers is not in the document]
Comment (smallhive) [reply spec/rest.yaml]: Of course not, it’s just a desperate step

File: spec/rest.yaml (hunk 1 above)
Comment (smallhive) [reply spec/rest.yaml]: I agree, but if the parameter is not defined in the request directly, the cogeneration tool generates not-compilable code

**Comment by smallhive** [comment]
> Have you checked that the test `Test Put over REST, Get over REST with headers` from `neofs-testcases` works with attributes `{'cat%jpeg': 'cat%jpeg'}`? It was marked as skipped earlier.

It is a good question. I tried this branch with the test. The test is green!

File: spec/rest.yaml (hunk 1 above)
Comment (tatiana-nspcc) [reply spec/rest.yaml]: Sadly okay

**Comment by tatiana-nspcc** [comment]
That's good. Maybe you can activate this test in the testcases repository and also resolve the issues of the failing automated test here?

**Comment by smallhive** [comment]
Eventually, we decided with @roman-khimov not to move forward with these changes

**Pull request closed without merging**